### PR TITLE
zh: update docs/concepts/observability

### PR DESCRIPTION
**Pull request by my-git9**
Please provide a description for what this PR is for.

Update and improve /zh/docs/concepts/observability.md
The en docs updated part:
image

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by windsonsea** [comment]
Thanks. Could you help to check the following lines that are required to sync with the en upstream too:

- Line 61
- Line 114
- Line 122
- Line 124
- LIne 128

**Comment by my-git9** [comment]
> Thanks. Could you help to check the following lines that are required to sync with the en upstream too:
> 
> * Line 61
> * Line 114
> * Line 122
> * Line 124
> * LIne 128

Updated

File: content/zh/docs/concepts/observability/index.md [hunk 1]
```diff
@@ -121,10 +121,10 @@ Istio 可以以一组可配置的格式集生成服务流量的访问日志，
 
 访问日志可以在本地生成，或者导出到自定义的后端基础设施，包括 [Fluentd](/zh/docs/tasks/observability/logs/fluentd/)。
 
-更多关于访问日志的内容在[收集日志](/zh/docs/tasks/observability/logs/collecting-logs/)和[获取 Envoy 服务日志](/zh/docs/tasks/observability/logs/access-log/)任务中提供。
+Istio 可以以一组可配置的格式为服务流量生成访问日志，使操作员可以完全控制日志记录的方式、内容、时间和地点。有关更多信息，请参阅[获取 Envoy 的访问日志]((/zh/docs/tasks/observability/logs/access-log/))。
```
Comment (windsonsea) [inline content/zh/docs/concepts/observability/index.md:124]:
```suggestion
Istio 能够以一组可配置的格式为服务流量生成访问日志，使操作员可以完全控制日志记录的方式、内容、时间和地点。有关更多信息，请参阅[获取 Envoy 的访问日志](/zh/docs/tasks/observability/logs/access-log/)。
```

File: content/zh/docs/concepts/observability/index.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ Istio 支持很多追踪系统，包括 [Zipkin](/zh/docs/tasks/observability/di
 
 更多关于 Istio 分布式追踪的信息可以在[分布式追踪 FAQ](/zh/about/faq/distributed-tracing/) 中找到。
 
-Istio 为一个请求生成的分布式追踪数据：
+Istio 为单个请求生成的分布式跟踪示例：
```
Comment (windsonsea) [inline content/zh/docs/concepts/observability/index.md:112]:
```suggestion
Istio 为单个请求生成的分布式追踪示例：
```
This page has 7 追踪 now. Both has the same meaning. It's better to follow the current style. Thanks.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**